cluster/charts/rook-ceph/templates/resources.yaml

```diff
@@ -32,6 +32,8 @@ spec:
                   type: boolean
                 urlPrefix:
                   type: string
+                port:
+                  type: integer
```

Comment:
if you add a minimum and maximum to the crd validation here, you won't need to worry about invalid ports in the code.